druid/Dockerfile

```diff
@@ -32,8 +32,26 @@ RUN curl -L https://repo.stackable.tech/repository/packages/druid/apache-druid-$
     # Install the Prometheus emitter extension. This bundle contains the emitter and all jar dependencies.
     curl https://repo.stackable.tech/repository/packages/druid/druid-prometheus-emitter-${PRODUCT}.tar.gz | tar -xzC /stackable/druid/extensions
 
-# Mitigate CVE-2021-44228 (Log4Shell)
+# ===
+# Mitigation for CVE-2021-44228 (Log4Shell)
+# This variable is supported as of Log4j version 2.10 and
+# disables the vulnerable feature
 ENV LOG4J_FORMAT_MSG_NO_LOOKUPS=true
 
+# For earlier versions this script removes the .class file that contains the
+# vulnerable code.
+# TODO: This can be restricted to target only versions which do not honor the environment
+#   varible that has been set above but this has not currently been implemented
+COPY shared/log4shell.sh /bin
+RUN /bin/log4shell.sh /stackable/druid-${PRODUCT}
+
+# Ensure no vulnerable files are left over
+# This will currently report vulnerable files being present, as it also alerts on
+# SocketNode.class, which we do not remove with our scripts.
+# Further investigation will be needed whether this should also be removed.
+COPY shared/log4shell_1.1.2-log4shell_Linux_x86_64 /bin/log4shell_scanner
```

Comment:
The file `shared/log4shell_1.1.2-log4shell_Linux_x86_64` is missing.

Reply:
added